scale/src/commonMain/kotlin/Max.kt

```diff
@@ -0,0 +1,12 @@
+package com.juul.krayon.scale
+
+public inline fun <I, O : Comparable<O>> List<I>.max(
```

Comment:
To be more general, should this be changed to?:

```suggestion
public inline fun <I, O : Comparable<O>> Collection<I>.max(
```

Reply:
Seems pretty reasonable

Reply:
#75